src/control-plane/backend/lambda/api/model/metadata.ts

```diff
@@ -60,8 +60,10 @@ export interface IMetadataEventParameter {
   platform: MetadataPlatform[];
   readonly parameterType: MetadataParameterType;
   readonly valueType: MetadataValueType;
-  valueEnum: IMetadataAttributeValue[];
+  valueEnum: string[];
```

Comment:
Let's remove the values and keep the array of `IMetadataAttributeValue` only.

Reply:
removed from response